tests/Unit_Validator.py

```diff
@@ -0,0 +1,16 @@
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
+import astropy.units
+
+class Unit_Validator():
+
+	def check_unit(self, unit_str):
+		try:
+			return astropy.units.Quantity("1 " + unit_str)
+		except ValueError:
+			return "ValueError: " + unit_str + " is not a valid unit."
+		except:
+			return "UnknownError: " + unit_str
```

Comment:
This test will do nothing because it has no method whose name starts with `test`. Is there some code that should call check_unit?

Reply:
Unit_Validator isn't a test, it's a utility called by test_Units.py.  I guess the check_unit function should go into test_utils.py?